doc/getting-started/getting-started.rst

```diff
@@ -378,121 +389,155 @@ their attributes, and the resources they have access to.
+
+   a. We recommend using nvm to manage your Node.js runtime. It allows you to
+      switch versions and update Node.js easily.
+
+      .. code-block:: bash
+
+         curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
+
+   #. In the new terminal, install the latest version of Node.js and npm:
+
+      .. code-block:: bash
+
+         source ~/.bashrc
+         nvm install node --latest-npm
+         nvm use node
+
+#. Install and upgrade Yarn:
 
    .. code-block:: bash
 
-      python3 acrn_configurator.py
+      npm install --global yarn
 
-#. Your web browser should open the website `<http://127.0.0.1:5001/>`__
-   automatically, or you may need to visit this website manually.
-   The ACRN Configurator is supported on Chrome and Firefox.
+#. Install rustup, the official installer for Rust:
 
-#. Click the **Import Board XML** button and browse to the board configuration
-   file ``my_board.xml`` previously generated. When it is successfully
-   imported, the board information appears.
-   Example:
+   .. code-block:: bash
 
-   .. image:: ./images/gsg_config_board.png
-      :class: drop-shadow
+      curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 
-#. Generate the scenario configuration file:
+   At this prompt, type ``1`` and press :kbd:`Enter`.
 
-   a. Click the **Scenario Settings** menu on the top banner of the UI and
-      select **Load a default scenario**. Example:
+   .. code-block:: console
 
-      .. image:: ./images/gsg_config_scenario_default.png
-         :class: drop-shadow
+      1) Proceed with installation (default)
+      2) Customize installation
+      3) Cancel installation
+      >1
 
-   #. In the dialog box, select **shared** as the default scenario setting and
-      then click **OK**. This sample default scenario defines six User VMs.
+#. Configure the current shell:
 
-      .. image:: ./images/gsg_config_scenario_load.png
-         :class: drop-shadow
+   .. code-block:: bash
 
-   #. The scenario's configurable items appear. Feel free to look through all
-      the available configuration settings used in this sample scenario. This
-      is where you can change the sample scenario to meet your application's
-      particular needs. But for now, leave them as they're set in the
-      sample.
+      source $HOME/.cargo/env
 
-   #. Click the **Export XML** button to save the scenario configuration file
-      that will be
-      used in the build process.
+#. Install ACRN Configurator dependencies:
 
-   #. In the dialog box, keep the default name as is. Type
-      ``/home/<username>/acrn-work`` in the Scenario XML Path field. In the
-      following example, ``acrn`` is the username. Click **Submit** to save the
-      file.
+   .. code-block:: bash
 
-      .. image:: ./images/gsg_config_scenario_save.png
-         :class: drop-shadow
+      cd ~/acrn-work/acrn-hypervisor/misc/config_tools/configurator
+      python3 -m pip install -r requirements.txt
```

Comment:
Any particular reason we're not using `pip3` directly here? I.e. `pip install -r requirements.txt`

Reply:
@gvancuts Not sure, this command came from Eng team. Should I change it to ``sudo pip3 install -r requirements.txt``?

Reply:
It looks like this is in fact the most robust mechanism to use - see also https://github.com/projectacrn/acrn-hypervisor/pull/7358#discussion_r858672890

Let's keep it as is (and perhaps we should update all `pip install ...` commands to this form (`python3 -m pip install...`) throughout our documentation.